CODE/Sources/PennySHARED/Models/S3AutoPingItems.swift

```diff
@@ -0,0 +1,36 @@
+
+public struct S3AutoPingItems: Codable {
+    
```

Comment:
maybe we want to add something like REGEX in the future?! this enums makes it nice and possible